src/stylesheets/components/_nav.scss

```diff
@@ -2,13 +2,14 @@
 
 .pillbar {
   display: none;
-  
+
   @media ($screen-sm-up) {
     display: flex;
     font-family: $main-font;
     height: 35px;
     margin: 55px auto;
-    width: 40vw;
+    transition: width 0.5s;
```

Comment:
When using transition, it's generally recommended to try to stick to transitioning properties that are hardware accellerated, e.g. opacity and transform.  I'd recommend sticking a pseudo elements at the center of and underneath the pillbar, set its width to something like 100px, and then do a scaleX transition on that pseudo element to get the "expanding" effect.